app/views/hq/orphans/_form.html.erb

```diff
@@ -0,0 +1,151 @@
+      <h3 class="panel-title">Orphan Details</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :name %>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :date_of_birth %>
+        <%= f.text_field :date_of_birth, class: "form-control date-picker", value: @orphan.date_of_birth %>
+      </div>
+      <div class="form-group">
+        <%= f.label :gender %>
+        <%= f.select :gender,
+          Settings.lookup.gender, {}, class: "form-control" %>
+      </div>
+      <%# Health status %>
+      <%# Schooling status %>
+      <div class="form-group">
+        <%= f.label :orphan_status_id %>
+        <%= f.collection_select :orphan_status_id,
+          @orphan_statuses, :id, :name, {}, 
+					{class: "form-control" , disabled: !f.object.new_record?} %>
+      </div>
+      <div class="form-group">
+        <%= f.label :orphan_sponsorship_status_id %>
+        <%= f.collection_select :orphan_sponsorship_status_id,
+          @orphan_sponsorship_statuses, :id, :name, {}, 
+					{class: "form-control", disabled: !f.object.new_record?} %>
+      </div>
+      <div class="form-group">
+        <%= f.label :priority %>
+        <%= f.select :priority, 
+          %w(Normal High), {}, class: "form-control" %>
+      </div>
+    </div>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Parents' Details</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :father_given_name %>
+        <%= f.text_field :father_given_name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :family_name %>
+        <%= f.text_field :family_name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :father_alive %>
+        <%= f.check_box :father_alive %> 
+      </div>
+      <div class="form-group">
+        <%= f.label :mother_name %>
+        <%= f.text_field :mother_name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :mother_alive %>
+        <%= f.check_box :mother_alive %>
+      </div>
+      <div class="form-group">
+        <%= f.label :father_is_martyr %>
+        <%= f.check_box :father_is_martyr %>
+      </div>
+    </div>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Original Address</h3>
+    </div>
+    <%= f.fields_for :original_address do |builder| %>
+			<div class="panel-body">
+				<div class="form-group">
+					<%= builder.label :city %>
+					<%= builder.text_field :city, class: "form-control" %>
+				</div>
+				<div class="form-group">
+					<%= f.label :province %>
+					<%= f.collection_select :original_address,
+					 @provinces, :id, :name, {},  {class: "form-control", disabled: !f.object.new_record?} %>
```

Comment:
Address provinces are not used by the app in any way and can be changed by users any time without danger of invalidating data. The fields here and on L112 should be enabled.